### ISSUE TITLE
confusion

### ISSUE THREAD

**Issue by IwGrin** [Justherozen/ProMix#4]
hi there,
老师请问，噪声的pt文件在哪里呢？

**Comment by IwGrin** [comment]
why warmup net1 but not net2.(conf_penalty(outputs) only computes net1), L is the loss of net1 and net2 and plus the penalty from net1, then L.backword, then optimizer.step()?
if you want train these two net simultaneously, why not L plus penalty from net2?

**Comment by Justherozen** [comment]
> 
please refer to https://github.com/UCSC-REAL/cifar-10-100n to obtain the noise file.

**Comment by Justherozen** [comment]
> 

Sorry for this confusion. The two peer networks are trained in the parallel and both need to be optimized through the penalty loss when facing the asymmetric noise. We have revised and updated our code.
